packages/vm/README.md

```diff
@@ -14,14 +14,16 @@ Implements Ethereum's VM in Javascript.
 
 The VM currently supports the following hardfork rules:
 
+- `Chainstart` (a.k.a. Frontier) (v5, UNRELEASED)
+- `Homestead` (v5, UNRELEASED)
+- `TangerineWhistle` (v5, UNRELEASED)
+- `SpuriousDragon` (v5, UNRELEASED)
 - `Byzantium`
 - `Constantinople`
 - `Petersburg` (default)
```

Comment:
Just discovered here with some shuddering surprise that we are still on `Petersburg` with the VM (re-checked in the code as well). We should move this to `Istanbul` before a release, have added this to the VM v5 release TODO list #681 (see reasoning there why not `MuirGlacier).

Comment is somewhat review-unrelated though.